src/Mod/PartDesign/CMakeLists.txt

```diff
@@ -55,9 +55,14 @@ set(PartDesign_TestScripts
     PartDesignTests/TestThickness.py
     PartDesignTests/TestTopologicalNamingProblem.py
     PartDesignTests/TestInvoluteGear.py
-    PartDesignTests/TestHelix.py
```

Comment:
Is this removal of TestHelix intentional?

Reply:
Yes. It was included twice